content/3-lifecycle/1-on-mount/lit/page-title.js

```diff
@@ -0,0 +1,17 @@
+import { LitElement, html } from 'lit';
+import { customElement, state } from 'lit/decorators.js';
+
+@customElement('page-title')
+export class PageTitle extends LitElement {
+	@state()
+	pageTitle = '';
+
+	connectedCallback() {
```

Comment:
Using constructor: https://lit.dev/docs/v1/components/events/#add-event-listeners-in-the-constructor
The `constructor` syntax looks cleaner to me. What do you think ?

Reply:
I chose `connectedCallback` here since it more closely resembles the concept of an `onMount` (since it gets called when "mounted" to the document). `constructor` gets called when the element is _created_.

E.g. `const element = document.createElement('page-title')` will call the constructor even if it hasn't mounted yet. Using the constructor could mean that by the time the element gets "mounted" the `pageTitle` state can be out of date.